q2_types/genome_data/_methods.py

```diff
@@ -12,7 +12,34 @@
 import numpy as np
 from qiime2.util import duplicate
 
-from q2_types.genome_data import SeedOrthologDirFmt, OrthologAnnotationDirFmt
+from q2_types.genome_data import (SeedOrthologDirFmt, OrthologAnnotationDirFmt,
+                                  LociDirectoryFormat)
+
+
+def collate_loci(loci: LociDirectoryFormat) -> LociDirectoryFormat:
```

Comment:
Hey @DorielaGrabocka, could you please simplify these loops to resemble the other collating actions we have in this plugin? There is one example in this file, at the bottom. Unless you had a reason to do it this way?